src/qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -41,19 +43,24 @@ class OutcomeDeclarationMarshaller extends VariableDeclarationMarshaller
      * Marshall an OutcomeDeclaration object into a DOMElement object.
      *
      * @param QtiComponent $component An OutcomeDeclaration object.
+     *
      * @return DOMElement The according DOMElement object.
-     * @throws MarshallerNotFoundException
      * @throws MarshallingException
+     *
+     * @throws MarshallerNotFoundException
      */
-    protected function marshall(QtiComponent $component)
+    protected function marshall(QtiComponent $component): DOMElement
     {
         $element = parent::marshall($component);
         $version = $this->getVersion();
 
         // deal with views.
         // !!! If $arrayViews contain all possible views, it means that the treated
         // !!! outcome is relevant to all views, as per QTI 2.1 spec.
-        if (Version::compare($version, '2.1.0', '>=') === true && !in_array($component->getViews()->getArrayCopy(), View::asArray())) {
+        if (Version::compare($version, '2.1.0', '>=') === true && !in_array(
+                $component->getViews()->getArrayCopy(),
+                View::asArray()
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?